lib/public/components/common/table/headers.js

```diff
@@ -36,7 +37,7 @@ export const headers = (columns, models) => {
         const classes = [size || null, column.classes || null];
         const attributes = {
             scope: 'col',
-            title: name.toLocaleUpperCase(),
+            title: column.information ? '' : name.toLocaleUpperCase(),
```

Comment:
Can you try if you have the same result with undefined or null? I think the intention would be more clear (if you can even add a small comment it would be perfect)